packages/adapter-interfaces/CHANGELOG.md

```
@@ -4,7 +4,10 @@
 
 ### Minor Changes
 
+<<<<<<< HEAD
```

markdownlint-cli2 (0.17.2)

Heading style
Expected: atx; Actual: setext

(MD003, heading-style)